velox/connectors/hive/tests/HiveConnectorTest.cpp

```diff
@@ -77,7 +89,7 @@ TEST_F(HiveConnectorTest, makeScanSpec_requiredSubfields_multilevel) {
   validateNullConstant(*values->childByName("c0c1c1"), *BIGINT());
 }
 
-TEST_F(HiveConnectorTest, makeScanSpec_requiredSubfields_mergeFields) {
```

Comment:
I would keep these underscores as they make the names easier to read

Reply:
Does this follow our coding standard? We use camel naming convention?